model.js

```diff
@@ -67,3 +81,45 @@ function formatFeature(apt) {
 function dateFormat(date) {
   return new Date(parseInt(date, 10) * 1000).toISOString();
 }
+
+// This next two functions could be wrapped in their own NPM that providers could require and use
+/**
+ * Format features and add an ArcGIS compliant OBJECTID to the idField
+ * @param {*} list list of feature to receive formatting
+ * @param {*} formatFunction the function that formats list items
+ */
+function createFeaturesWithArcGISCompliantIDs(list, formatFunction) {
+  const MAX_OBJECTID = 2147483647
```

Comment:
I think this is probably overly complicated. I'd do this in the most simple way possible and document the gotchas.